docs/content/guides/dagster/managing-ml.mdx

```diff
@@ -129,15 +129,23 @@ def conditional_machine_learning_model(context: AssetExecutionContext):
         AssetKey(["conditional_machine_learning_model"])
     )
     if materialization is None:
-        yield Output(reg, metadata={"model_accuracy": reg.score(X_test, y_test)})
+        yield Output(reg, metadata={"model_accuracy": float(reg.score(X_test, y_test))})
 
     else:
-        previous_model_accuracy = materialization.asset_materialization.metadata[
-            "model_accuracy"
-        ]
+        previous_model_accuracy = None
```

Comment:
@odette-elementl  interested in your sign-off on these changes. Most of it is type casting so that the comparisons are between `float` types instead of `float` and `MetadataValue` which pyright doesn't think is valid